example/common/flat_write_stream.hpp

```diff
@@ -464,38 +434,56 @@ namespace beast {
         {
             Handler h_;
             flat_write_stream& s_;
-            boost::asio::const_buffers_1 cb1_;
-            std::shared_ptr<char> ptr_;
+            ConstBufferSequence const& b_;
```

Comment:
You can't save a reference to the buffer sequence because it will go out of scope. You either need to make a copy, or just do what you need to do with it in the constructor (I think you're already doing this). You don't need `b_`.